lib/kb_trimmomatic/kb_trimmomaticImpl.py

```diff
@@ -202,6 +202,32 @@ def parse_trimmomatic_steps(self, input_params):
 
         return parameter_string
 
+    def _save_RNASeqSampleSet(self, items, wsName, output_SampleSet_name, reads_desc_ext,
+                              single_reads):
+
+        print ('Start saving RNASeqSampleSet object')
+        workspace_id = self.dfu.ws_name_to_id(wsName)
+        Library_type = 'SingleEnd' if single_reads else ''
```

Comment:
PairedEnd is a valid option for the Library type, it should be handled here. And it should come from whatever the input Library_type was for the input RNASeqSampleSet object.

Reply:
👍